### PR TITLE
Added Carbon Black Panther pipeline

### PR DESCRIPTION
**Pull request by melenevskyi**
Added Carbon Black Panther pipeline for mapping Sigma to Carbon Black fields

### CODE REVIEW

File: sigma/pipelines/panther/sdyaml_transformation.py [hunk 1]
```diff
@@ -53,7 +53,10 @@ def _detect_log_types(self, rule: SigmaRule) -> [str]:
...
 
         cli_context = click.get_current_context(silent=True)
-        if cli_context and "crowdstrike_fdr" in cli_context.params["pipeline"]:
-            log_types.append("Crowdstrike.FDREvent")
+        if cli_context:
+            if "crowdstrike_fdr" in cli_context.params["pipeline"]:
+                log_types.append("Crowdstrike.FDREvent")
 
+            if "carbon_black_panther" in cli_context.params["pipeline"]:
+                log_types.append("CarbonBlack.EndpointEvent")
```
Comment (egibs) [inline sigma/pipelines/panther/sdyaml_transformation.py:61]: Will we support the other Carbon Black log types eventually?

As of right now, PAT supports `EndpointEvent` and three additional log types:
- `CarbonBlack.AlertV2`
- `CarbonBlack.Audit`
- `CarbonBlack.EndpointEvent`
- `CarbonBlack.WatchlistHit`

Comment (melenevskyi) [reply sigma/pipelines/panther/sdyaml_transformation.py:61]: cc @arielkr256

Comment (egibs) [reply sigma/pipelines/panther/sdyaml_transformation.py:61]: It's not a blocker for this PR but I was curious. 😅

File: sigma/pipelines/panther/panther_sdyaml_pipeline.py [hunk 2]
```diff
@@ -24,6 +23,18 @@ def logsource_linux():
     return LogsourceCondition(product="linux")
 
 
+def logsource_file_event():
```
Comment (egibs) [inline sigma/pipelines/panther/panther_sdyaml_pipeline.py:26]: Nice -- this is cleaner 👍🏻

**Review by egibs — APPROVED** [review]
LGTM!

File: sigma/pipelines/panther/sdyaml_transformation.py (hunk 1 above)
Comment (arielkr256) [reply sigma/pipelines/panther/sdyaml_transformation.py:61]: This is correct, Sigma rules will only look at `CarbonBlack.EndpointEvent`

**Review by arielkr256 — APPROVED** [review]
Looks awesome!

**Pull request merged**